### PR TITLE
Panel model

### PR DESCRIPTION
**Pull request by wct9667**
# Description
Added a `Panel` Model and two queries, one for creating and other for getting based on `id`.

## Model - Panel
| id | image | index | panel_set_id | created_at | updated_at |
| -------- | ------- | -------- | ------- | -------- | ------- |
| `INTEGER`  | `STRING` | `SMALLINT`| `INTEGER` | `TIMESTAMP` | `TIMESTAMP` |

## `PanelService.ts` 
Has two queries: 
### `createPanel` 
creates a panel given a `image`, `index` and `panel_set_id`, It returns a object with
 `{id, image, index, panel_set_id }` once created.
### `getPanel` 
gets a panel given a `id`.  It returns a object with
 `{image, index, panel_set_id }`.

### CODE REVIEW

File: src/database/services/PanelService.ts [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+
+/**
+ * Perform queries on the 'Panels' table
+ */
+class PanelService {
+
+    /**
+     * Create a new Panel}
+     * @param {} newPanel
+     */
```
Comment (4rt3ry) [inline src/database/services/PanelService.ts]: funny typo

File: src/database/services/PanelService.ts [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+        });
+        return {id, image, index, panel_set_id };
+    }
+
+    /**
+     * Authenticate a panel before retrieve the panel's data
+     * @param id 
+     * @returns 
+     */
+    static async getPanel(id: number) {
```
Comment (4rt3ry) [inline src/database/services/PanelService.ts:46]: I imagine there will be more ways to get a panel in the future, but getting by id is fine for now. 
Also, I think you should specify a return type (I would suggest creating a `PanelInfo {}` interface)

File: src/database/services/PanelService.ts [hunk 3]
```diff
@@ -0,0 +1,47 @@
...
+/**
+ * Perform queries on the 'Panels' table
+ */
+class PanelService {
+
+    /**
+     * Create a new Panel}
+     * @param {} newPanel
+     */
+    static async createPanel(newPanel: PanelConfig){
```
Comment (4rt3ry) [inline src/database/services/PanelService.ts:32]: add a return type

File: src/database/services/PanelService.ts (hunk 2 above)
Comment (wct9667) [reply src/database/services/PanelService.ts:46]: yeah, I just did what was on the projects board. We can discuss with Travis what other queries are useful today.

File: src/database/services/PanelService.ts [hunk 4]
```diff
@@ -0,0 +1,60 @@
...
+            index: newPanel.index,
+            panel_set_id: newPanel.panel_set_id,
+        });
+        return {id, image, index, panel_set_id } as PanelInfoCreate;
+    }
+
+    /**
+     * Authenticate a panel before retrieve the panel's data
+     * @param id 
+     * @returns
```
Comment (nca6109) [inline src/database/services/PanelService.ts]: Return type again

**Review by nca6109 — APPROVED** [review]
Looks good to me

**Pull request merged**